assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/clients/DataCiteClientTest.java

```diff
@@ -12,12 +12,12 @@
 import java.net.URISyntaxException;
 import java.util.Map;
 import no.unit.nva.doi.datacite.clients.exception.ClientException;
-import no.unit.nva.doi.datacite.clients.models.Doi;
 import no.unit.nva.doi.datacite.config.DataCiteConfigurationFactory;
 import no.unit.nva.doi.datacite.config.DataCiteConfigurationFactoryForSystemTests;
 import no.unit.nva.doi.datacite.mdsclient.DataCiteMdsConnection;
 import no.unit.nva.doi.datacite.mdsclient.DataCiteMdsConnectionFactory;
 import no.unit.nva.doi.datacite.models.DataCiteMdsClientSecretConfig;
+import no.unit.nva.doi.models.Doi;
```

Comment:
Uses `publication-doi-type` module now. Currently with publication service due to it also is convenient to be together with `doi-commons`.